Dockerfile

```diff
@@ -2,20 +2,10 @@ FROM node:7.0.0
 
 COPY . deck/
 
-RUN useradd -ms /bin/bash node
-
-RUN chown -R node deck
-
-RUN chown -R node /usr/
-
-ENV HOME /home/node
-
-USER node
-
 WORKDIR deck
 
-RUN rm -rf .git
+RUN npm install --production
 
-RUN npm install
+RUN mv ./node_modules ./node_modules.tmp && mv ./node_modules.tmp ./node_modules && npm install
```

Comment:
should probably just nuke node modules in the container instead of having the image being large for no reason